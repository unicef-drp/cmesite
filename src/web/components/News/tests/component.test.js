import React from 'react';
import { shallow } from 'enzyme';
import Component from '../component';

describe('News | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const root = <Component />;

      const wrapper = shallow(root);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
