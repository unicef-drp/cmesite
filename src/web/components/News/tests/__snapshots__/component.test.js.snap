// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News | component render should match snapshot 1`] = `
<News
  classes={
    Object {
      "card": "News-card-3",
      "container": "News-container-2",
      "media": "News-media-4",
      "typo": "News-typo-5",
      "wrapper": "News-wrapper-1",
    }
  }
  news={Array []}
/>
`;
